### ISSUE TITLE
Sol is a full charge instead of a 6 charge

### ISSUE THREAD

**Issue by Zamiell** [UnderscoreKilburn/repentance-issues#864]
In previous patches, other 12-charge methods in the game have been reduced to 6.
Thus, Sol should be changed in the same way for consistency.

**Comment by TiredFoks** [comment]
I do not agree, Sol is not really that good to begin with, and making an item worse just for consistency sake isn't a good idea.
